src/order_logger/loggable_order.go

```diff
@@ -6,18 +6,23 @@ import (
 	"github.com/HyperloopUPV-H8/Backend-H8/vehicle/models"
 )
 
-type LoggableOrder models.PacketUpdate
+type LoggableOrder models.Order
 
 func (lo LoggableOrder) Id() string {
-	return fmt.Sprint(lo.Metadata.ID)
+	return fmt.Sprint(lo.ID)
 }
 
 func (lo LoggableOrder) Log() []string {
-	log := []string{fmt.Sprint(lo.Metadata.ID)}
+	return []string{fmt.Sprintf("[FROM GUI]: %d %v", lo.ID, lo.Fields)}
```

Comment:
I don't like this format for the log file, since we use CSV we should try to log it like that. This just creates one column which is difficult to process later, instead, try to split the information into columns. For our data what I thing is best is to have the following columns: timestamp, from, to, seqNum, id and values. In this case you don't have all the information, that is fine, just fill the timestamp (with `time.Now()`), from (set it to `control` or something like that), id and values and leave the others empty.